src/styles/Global.tsx

```diff
@@ -102,6 +102,7 @@ export const GlobalStyle = createGlobalStyle`
     transition: all 0.2s;
     &:hover {
       color: ${props => props.theme.colors.text};
+      transform: scale3d(1.05, 1.05, 1);
```

Comment:
Is this animation mentioned in the design? If you make all the <a> tags animated, it will affect others. So please quote the section showing why you added this animation.